packages/auth/src/userAuth.js

```diff
@@ -7,11 +7,38 @@ import jwt from 'jsonwebtoken';
 import { UnauthenticatedError } from '@tupaia/utils';
 import { getJwtToken } from './security';
 
+const ACCESS_TOKEN_EXPIRY_SECONDS = 15 * 60; // User's access expires every 15 mins
+
+export const constructAccessToken = ({ userId, apiClientUserId, refreshToken }) => {
```

Comment:
Moved this out of `meditrak-server` so that creating/decrypting accessTokens is done side by side.

Reply:
Nice!